Add Chess

biesiadamich opened the pull request:
Updated awesome-chess.

[Insert URL to the list here]
https://github.com/mbiesiad/awesome-chess#readme

[Explain what this list is about and why it should be included here]
Comprehensive, professional awesome list about Chess.
Chess-related awesome list.

Updated - the current link looks stale (see the issues).

Reviewed PRs:
#1788 #1784
By submitting this pull request I confirm I've read and complied with the below requirements 🖖

Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.
Requirements for your pull request

    Don't waste my time. Do a good job, adhere to all the guidelines, and be responsive.
    You have to review at least 2 other open pull requests.
    Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
    You have read and understood the instructions for creating a list.
    This pull request has a title in the format Add Name of List.
        ✅ Add Swift
        ✅ Add Software Architecture
        ❌ Update readme.md
        ❌ Add Awesome Swift
        ❌ Add swift
        ❌ Adding Swift
        ❌ Added Swift
    Your entry here should include a short description about the project/theme of the list. It should not describe the list itself. The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
        ✅ - [iOS](…) - Mobile operating system for Apple phones and tablets.
        ✅ - [Framer](…) - Prototyping interactive UI designs.
        ❌ - [iOS](…) - Resources and tools for iOS development.
        ❌ - [Framer](…)
        ❌ - [Framer](…) - prototyping interactive UI designs
    Your entry should be added at the bottom of the appropriate category.
    The suggested Awesome list complies with the below requirements.

Requirements for your Awesome list

    Has been around for at least 30 days.
    That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
    Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it.
    Run awesome-lint on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please report it.
    Includes a succinct description of the project/theme at the top of the readme. (Example)
        ✅ Mobile operating system for Apple phones and tablets.
        ✅ Prototyping interactive UI designs.
        ❌ Resources and tools for iOS development.
        ❌ Awesome Framer packages and tools.
    It's the result of hard work and the best I could possibly produce.
    If you have not put in considerable effort into your list, your pull request will be immediately closed.
    The repo name of your list should be in lowercase slug format: awesome-name-of-list.
        ✅ awesome-swift
        ✅ awesome-web-typography
        ❌ awesome-Swift
        ❌ AwesomeWebTypography
    The heading title of your list should be in title case format: # Awesome Name of List.
        ✅ # Awesome Swift
        ✅ # Awesome Web Typography
        ❌ # awesome-swift
        ❌ # AwesomeSwift
    Non-generated Markdown file in a GitHub repo.
    The repo should have awesome-list & awesome as GitHub topics. I encourage you to add more relevant topics.
    Not a duplicate. Please search for existing submissions.
    Only has awesome items. Awesome lists are curations of the best, not everything.
    Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
    Includes a project logo/illustration whenever possible.
        Either centered, fullwidth, or placed at the top-right of the readme. (Example)
        The image should link to the project website or any relevant website.
        The image should be high-DPI. Set it to maximum half the width of the original image.
    Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
    Includes the Awesome badge.
        Should be placed on the right side of the readme heading.
            Can be placed centered if the list has a centered graphics header.
        Should link back to this list.
    Has a Table of Contents section.
        Should be named Contents, not Table of Contents.
        Should be the first section in the list.
        Should only have one level of nested lists, preferably none.
    Has an appropriate license.
        We strongly recommend the CC0 license, but any Creative Commons license will work.
            Tip: You can quickly add it to your repo by going to this URL: https://github.com/<user>/<repo>/community/license/new?branch=master&template=cc0-1.0 (replace <user> and <repo> accordingly).
        A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are WTFPL and Unlicense.
        Place a file named license or LICENSE in the repo root with the license text.
        Do not add the license name or text to the readme. GitHub already shows the license name at the top of the repo.
        To verify that you've read all the guidelines, please comment on your pull request with just the word unicorn.
    Has contribution guidelines.
        The file should be named contributing.md. Casing is up to you.
    Has consistent formatting and proper spelling/grammar.
        The link and description are separated by a dash.
        Example: - [AVA](…) - JavaScript test runner.
        The description starts with an uppercase character and ends with a period.
        Consistent and correct naming. For example, Node.js, not NodeJS or node.js.
    Doesn't include a Travis badge.
    You can still use Travis for list linting, but the badge has no value in the readme.
    Doesn't include an Inspired by awesome-foo or Inspired by the Awesome project kinda link at the top of the readme. The Awesome badge is enough.

Go to the top and read it again.
Reopen #1791

Best wishes,

MrPeker commented:
Nice work, it seems good to me.

biesiadamich commented:
unicorn

sindresorhus commented:
Pretty much the same feedback as https://github.com/sindresorhus/awesome/pull/1802. Ensure you follow all the guidelines and try to improve the list further.

biesiadamich commented:
@sindresorhus thank you, improved.

sindresorhus commented on readme.md:
```diff
@@ -386,7 +386,7 @@
 - [Open Source Games](https://github.com/leereilly/games#readme)
 - [Unity](https://github.com/RyanNielson/awesome-unity#readme) - Game engine.
-- [Chess](https://github.com/hkirat/awesome-chess#readme)
+- [Chess](https://github.com/mbiesiad/awesome-chess#readme)
```
Since this is a new list, you need to include a description of what Chess is.

biesiadamich replied:
Done, added.

sindresorhus commented:
Entries in https://github.com/mbiesiad/awesome-chess#websites are missing descriptions.

sindresorhus commented:
> Includes a project logo/illustration whenever possible.
    Either centered, fullwidth, or placed at the top-right of the readme. (Example)
    The image should link to the project website or any relevant website.
    The image should be high-DPI. Set it to maximum half the width of the original image.

Should be easy to find some free stock art for Chess.

biesiadamich commented:
@sindresorhus good points, added.

sindresorhus commented:
Still multiple things not done. The list is also light on content. Please put more effort into it and go through the guidelines again.

The table on contents should not be a numbered list.

I found an invalid link.

The stock photo you picked doesn't even look like chess. There are so many good ones available... https://unsplash.com/s/photos/chess

I don't think the `National chess federations` section is very interesting. It should be at the bottom. The most interesting content should be first.

Do you play chess?

biesiadamich commented:
> 
> 
> Still multiple things not done. The list is also light on content. Please put more effort into it and go through the guidelines again.
> 
> The table on contents should not be a numbered list.
> 
> I found an invalid link.
> 
> The stock photo you picked doesn't even look like chess. There are so many good ones available... https://unsplash.com/s/photos/chess
> 
> I don't think the `National chess federations` section is very interesting. It should be at the bottom. The most interesting content should be first.
> 
> Do you play chess?

Yup, I'm professional chess player. Do you want to play?

biesiadamich commented:
Many of projects on your list have numbered list in ToC + awesome-linter said is OK. Changed.

Your invalid link is perhaps about issue created on awesome-linter as 'false-positive'. By the way, after few years many of links included in your list could be marked as dead links (moreover, a lot of links there are invalid now, because of time).

'The most interesting content should be first', agree. Changed.
Photo, changed.

angular-chess - you're right in general, but for some people it could be still useful.
'Why link to the Angular wrapper and not', the other resources.

Ok I see, but feeling vs reality.

Changed some items.

tobiasvl commented:
Seems like you've done a lot of work here! But the list seems kind of jumbled to me.

Your [CONTRIBUTING.md](https://github.com/mbiesiad/awesome-chess/blob/master/CONTRIBUTING.md) looks incomplete. The guidelines are placeholders, and the file doesn't explain what makes a chess resource "awesome".

There is no link to information about the "Belle" computer.

The "Organizations" section seems broken/incomplete. Should "FIDE" be a subsection of that one? What about the later section "National chess federations"?

What is the "Projects" section about? What kinds of projects? What makes them different from the other repos/projects in other sections?

The "Other" sections seems strange to me. What makes those entries, including links to seemingly random Wikipedia articles (Magnus Carlsen is self-explanatory to me since I know he's the World Champion, but perhaps not to others; and why is there a link to Hikaru Nakamura's page and not other chess players?), awesome?

biesiadamich commented:
Hi @tobiasvl Thanks for your feedback, I really appreciate it! I will consider that, thanks again!

sindresorhus commented:
Closing as I never got a response on the other list PR, and generally, I think you need to spend more effort on the lists and the PRs you make. You are free to submit again in some months if the lists have improved.
